<p>This rule prohibits the usage of CHAIN as the process control will never return to invoker.</p>
<h2>Noncompliant Code Example</h2>
<pre>CHAIN 'OFF'</pre>
<h2>Compliant Solution</h2>
<p><b>CHAIN</b> is not recommended as per T24 standards.</p>
<pre></pre>
