<p>This rule is used to check whether CRT has been used for GUIs.</p>
<h2>Noncompliant Code Example</h2>
<pre>CRT 'Hello World'</pre>
<h2>Compliant Solution</h2>
<p>Use Logger if necessary to capture the output with GUIs.</p>
<pre></pre>
<h2>Exceptions</h2>
<p>CRT holds good only for command line processing.</p>
