<p>This rule is used to check whether TXTINPUT has been used for GUIs.</p>
<h2>Noncompliant Code Example</h2>
<pre>CALL TXTINP (YTEXT,C2,L2,N1,T1)
EB.Display.Txtinp(Ytext, C2, L2, N1, T1)</pre>
<h2>Compliant Solution</h2>
<p>NA for GUIs.</p>
<pre></pre>
<h2>Exceptions</h2>
<p>TXTINPUT holds good only for command line processing.</p>
