<p>This rule is used to check whether PRINT has been used for GUIs.</p>
<h2>Noncompliant Code Example</h2>
<pre>PRINT var1</pre>
<h2>Compliant Solution</h2>
<p>Use Logger if necessary to capture the output with GUIs.</p>
<pre></pre>
<h2>Exceptions</h2>
<p>PRINT holds good only for command line processing.</p>
