<p>This rule is used to check whether INPUT has been used for GUIs.</p>
<h2>Noncompliant Code Example</h2>
<pre>INPUT var1</pre>
<h2>Compliant Solution</h2>
<p>NA for GUIs.</p>
<pre></pre>
<h2>Exceptions</h2>
<p>INPUT holds good only for command line processing.</p>
