<p>This rule restricts the usage of ABORT statement.</p>
<h2>Noncompliant Code Example</h2>
<pre>ABORT transaction</pre>
<h2>Compliant Solution</h2>
<p>Do not use <b>ABORT</b> statement, as it is handled by T24 transaction management.</p>
<pre></pre>
