<p>This rule makes it mandatory to remove the exclamation mark before HUSHIT as !HUSHIT is not supported in TAFJ.</p>
<h2>Noncompliant Code Example</h2>
<pre>CALL !HUSHIT(0)</pre>
<h2>Compliant Solution</h2>
<p>!(exclamation) should not be used before <b>HUSHIT</b>.</p>
<pre>CALL HUSHIT(0)</pre>
