<p>This rule prevents the usage of C functions as it is not recommended.</p>
<h2>Noncompliant Code Example</h2>
<pre>COMPILED.OR.NOT = CALLC JBASESubroutineExist('SONAR.PROG',PRG.INFO)</pre>
<h2>Compliant Solution</h2>
<p>Equivalent java function should be provided and it should be called directly instead of CALLC.</p>
<pre></pre>
